framework/src/media/audio/audio_manager.c

```diff
@@ -127,12 +127,17 @@ struct audio_device_config_s {
 struct audio_resample_s {
 	bool necessary;
 	uint8_t samprate_types;
-	unsigned int from;
-	unsigned int to;
+	unsigned int sample_src;
+	unsigned int sample_dst;
 	void *buffer;
 	unsigned int buffer_size;
 	float ratio;
 	src_handle_t handle;
+	uint8_t channel_src;		// channel info from an user
```

Comment:
src can be removed because tinyalsa provides apis.

Reply:
Ok, I'll modify it after basic functions are implemented.